datahub-frontend/play.gradle

```diff
@@ -19,15 +19,11 @@ dependencies {
 
   play externalDependency.jerseyCore
   play externalDependency.jerseyGuava
-  play externalDependency.mysqlConnector
-  play externalDependency.playJavaJdbc
```

Comment:
Can you get rid of this from top level `build.gradle` as well? I don't think anyone else is using  it other than frontend.

Reply:
Good point. Removed.